includes/bootstrap.php

```diff
@@ -159,6 +164,9 @@ function load(): void {
 
 	// Hook feature initialization to init.
 	add_action( 'init', __NAMESPACE__ . '\initialize_features' );
+	add_action( 'admin_init', __NAMESPACE__ . '\initialize_admin' );
```

Comment:
`admin_init` doesn't work here because when `Settings_Page::register()` runs, `admin_menu` has already fired. I tested with it attached to `init` and it works fine.

```suggestion
	add_action( 'init', __NAMESPACE__ . '\initialize_admin' );
```

Reply:
@mindctrl good catch handled it